app/assets/stylesheets/costs/costs_legacy.css

```diff
@@ -97,6 +102,24 @@ td.comment input {
   cursor: pointer;
 }
 
+.budget_fieldset{
+  margin: 8px 2px;
+}
+
+.budget_table_inputFields{
```

Comment:
I'd rather stick to a uniform casing strategy

* `snake_case`
* `camelCase`
* `dash-case`

`but_notMixed`.

Thinking about our other css classes it should probably be `budget-table--fieldset`, shouldn't it?